cg/apps/lims/api.py

```diff
@@ -217,6 +217,13 @@ def update_sample(self, lims_id: str, sex=None, application: str=None,
             lims_sample.udf[PROP2UDF['target_reads']] = target_reads
         lims_sample.put()
 
+    def update_project(self, lims_id: str, name=None):
+        """Update information about a project."""
+        lims_project = Project(self, id=lims_id)
+        if name:
+            lims_project.name = name
+        lims_project.put()
```

Comment:
I wonder if we should put the same thing back eventhough we seeminlgy haven't changed it. I'm a bit worried that retrieving it might set default values that weren't set before.

Reply:
we have the check on line 218         
`if microbial_order_obj.name == name:`
What do you suggest?

Reply:
```suggestion
            lims_project.put()
```

;)